File: AutoPowerModeSwitcher/SettingsDatabase.vb
Imports AutoPowerModeSwitcher.My

Public Class SettingsDatabase
    Private Sub Form2_Load(sender As Object, e As EventArgs) Handles MyBase.Load
       
    End Sub

    Private Sub CheckBox1_CheckedChanged(sender As Object, e As EventArgs) Handles CheckBox1.CheckedChanged
        If CheckBox1.Checked Then
            TopMost = true

            else
            topmost = false

        End If
    End Sub

    Private Sub Engine_Tick(sender As Object, e As EventArgs) Handles Engine.Tick
        Label1.Text = "TopMost= " & My.Settings.TopMost
        Label2.Text = "CheckInvertalEqualsFrequent= " & My.Settings.CheckInvertalEqualsFrequent
        Label3.Text = "PreferredPowerModeEqualsBalanced= " & My.Settings.PreferredPowerModeEqualsBalanced
        Label4.Text = "GamingPowerModeEquals70Freq= " & My.Settings.GamingPowerModeEquals70Freq
        Label5.Text = "TargetProcessToString= " & My.Settings.TargetProcess
        Label6.Text = "StartOnBootEnabled= " & My.Settings.StartOnBootEnabled
        Label7.Text = "StartMinimized= " & My.Settings.StartMinimized
        Label8.Text = "TopMost= " & My.Settings.TopMost
    End Sub
End Class

File: AutoPowerModeSwitcher/SettingsDatabase.cs
using System;
using AutoPowerModeSwitcher.My;

namespace AutoPowerModeSwitcher
{

    public partial class SettingsDatabase
    {
        public SettingsDatabase()
        {
            InitializeComponent();
        }
        private void Form2_Load(object sender, EventArgs e)
        {

        }

        private void CheckBox1_CheckedChanged(object sender, EventArgs e)
        {
            if (CheckBox1.Checked)
            {
                TopMost = true;
            }

            else
            {
                TopMost = false;

            }
        }

        private void Engine_Tick(object sender, EventArgs e)
        {
            Label1.Text = "TopMost= " + MySettingsProperty.Settings.TopMost;
            Label2.Text = "CheckInvertalEqualsFrequent= " + MySettingsProperty.Settings.CheckInvertalEqualsFrequent;
            Label3.Text = "PreferredPowerModeEqualsBalanced= " + MySettingsProperty.Settings.PreferredPowerModeEqualsBalanced;
            Label4.Text = "GamingPowerModeEquals70Freq= " + MySettingsProperty.Settings.GamingPowerModeEquals70Freq;
            Label5.Text = "TargetProcessToString= " + MySettingsProperty.Settings.TargetProcess;
            Label6.Text = "StartOnBootEnabled= " + MySettingsProperty.Settings.StartOnBootEnabled;
            Label7.Text = "StartMinimized= " + MySettingsProperty.Settings.StartMinimized;
            Label8.Text = "TopMost= " + MySettingsProperty.Settings.TopMost;
        }
    }
}